release notes/v0.47.0.md

```diff
@@ -75,12 +77,21 @@ _Format as `<number> <present_verb> <object>. <credit>`_:
 - [browser#1040](https://github.com/grafana/xk6-browser/pull/1040) fix the `context.clearCookies()` method to clear all cookies from the current browser context.
 - [browser#1031](https://github.com/grafana/xk6-browser/pull/1031) fix the `expires` field while adding cookies using the `context.addCookie()` method.
 - [browser#1039](https://github.com/grafana/xk6-browser/pull/1039) fix Goja conversions while adding and retrieving cookies.
+- [browser#1038](https://github.com/grafana/xk6-browser/pull/1038) Fix read/write race condition for edge case with remote browsers.
+- [browser#1034](https://github.com/grafana/xk6-browser/pull/1034) Fix `page.reload` & `page.setContent` to use the default navigation timeout over the default timeout.
```

Comment:
Is it a bug fix?

Reply:
Yeah, i think so. If a user used `page.reload` and `page.setContent` and had set the `defaultNavigationTimeout` then before this change the timeout wouldn't have worked as the user would expect. After the change it would work as they expected.